### PR TITLE
Rules 1.5 1.11

### PR DESCRIPTION
**Pull request by PrzemyslawTusinski**

### CODE REVIEW

File: rules/CIS-1.11_Ensure_IAM_password_policy_expires_passwords_within_90_days_or_less.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  IAM password policies can require passwords to be rotated or expired after a given
+  number of days. It is recommended that the password policy expire passwords after 90
+  days or less
+severity: high
+enabled: true
+sql: >
+  SELECT arn
+  FROM awsiamaccount
+  WHERE supplementaryConfiguration->'passwordPolicy' = 'null'
+    OR supplementaryConfiguration->'passwordPolicy'->'maxPasswordAge' <= '90';
```
Comment (stefansjfw) [inline rules/CIS-1.11_Ensure_IAM_password_policy_expires_passwords_within_90_days_or_less.yaml:16]: \> 90

File: rules/CIS-1.9_Ensure_IAM_password_policy_requires_minimum_length_of_14_or_greater.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  Password policies are, in part, used to enforce password complexity requirements. IAM
+  password policies can be used to ensure password are at least a given length. It is
+  recommended that the password policy require a minimum password length 14
+severity: high
+enabled: true
+sql: >
+  SELECT arn
+  FROM awsiamaccount
+  WHERE supplementaryConfiguration->'passwordPolicy' = 'null'
+    OR supplementaryConfiguration->'passwordPolicy'->'requireNumbers' = 'false';
```
Comment (stefansjfw) [inline rules/CIS-1.9_Ensure_IAM_password_policy_requires_minimum_length_of_14_or_greater.yaml:16]: query is wrong

File: rules/CIS-1.11_Ensure_IAM_password_policy_expires_passwords_within_90_days_or_less.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+description: >
+  IAM password policies can require passwords to be rotated or expired after a given
+  number of days. It is recommended that the password policy expire passwords after 90
+  days or less
+severity: high
+enabled: true
+sql: >
+  SELECT arn
+  FROM awsiamaccount
+  WHERE supplementaryConfiguration->'passwordPolicy'->'maxPasswordAge' = 'null'
```
[the remark this answers is not in the document]
Comment (PrzemyslawTusinski) [reply rules/CIS-1.11_Ensure_IAM_password_policy_expires_passwords_within_90_days_or_less.yaml:15]: No

[the remark this answers is not in the document]
Comment (PrzemyslawTusinski) [reply rules/CIS-1.11_Ensure_IAM_password_policy_expires_passwords_within_90_days_or_less.yaml:15]: Other rules either always have given field, or doesn't use > or < operator

**Review by stefansjfw — APPROVED** [review]

**Pull request merged**